frontend/src/types/issue.ts

```diff
@@ -45,17 +46,28 @@ export type UpdateSchemaDetail = {
   earliestAllowedTs: number;
 };
 
+export type UpdateSchemaGhostDetail = UpdateSchemaDetail & {
+  // empty by now
+  // more input parameters in the future
+};
+
 export type UpdateSchemaContext = {
   migrationType: MigrationType;
   updateSchemaDetailList: UpdateSchemaDetail[];
 };
 
+export type UpdateSchemaGhostContext = {
+  migrationType: MigrationType;
```

Comment:
I guess we don't have migration type for Gh-ost since all changes will be MIGRATE.

Reply:
Agree.